src/application/Types/PersonDetailsTypes.ts

```diff
@@ -0,0 +1,12 @@
+export interface PersonDetailsTypes {
```

Comment:
Why do we start using Types naming convention here? Why they are different?

Reply:
did you mean that why i changed type names here? if so because i needed to seperate "my own variable names" from the code benath it. and use this if i need anywhere else but this layer.